$NetBSD: patch-ab,v 1.3 2010/07/26 02:08:35 schnoebe Exp $

# make the spool path configurable by pkgsrc.
# make the pgsql back end settings visible.

--- example-config.xml.orig	2006-08-06 10:45:21.000000000 -0500
+++ example-config.xml
@@ -26,14 +26,13 @@
     <type>pgsql</type>
     <dbuser>muc</dbuser>
     <dbname>muc</dbname>
-    <!-- <dbpass>secret</dbpass> -->
-    <!-- <dbhostname>localhost</dbhostname> -->
-   
+    <dbpass>secret</dbpass>
+    <dbhostname>localhost</dbhostname>
   </backend>
   -->
 
   <!-- spool is the directory where filesystem based backends store data -->
-  <spool>/var/lib/jabberd/chat.localhost</spool>
+  <spool>@JABBER_SPOOLDIR@/chat</spool>
 
    <conference xmlns="jabber:config:conference">
     <sadmin>
